src/d3a/setup/d3a-settings.json

```diff
@@ -98,7 +98,6 @@
         10000
       ],
       "MIN_ALLOWED_SOC": 10,
-      "SELL_ON_MOST_EXPENSIVE_MARKET": false,
```

Comment:
This needs a small PR on d3a-interface, right?

Reply:
Hmmm, seems like I forgot to create the respective PR in the d3a-interface, doing it now, thanks!